source/assets/js/components/fef-a11y.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Some browsers do not fully support in-page links. While they may visually shift focus to the location
+ * of the target or named anchor for the “skip” link, they do not actually set keyboard focus to this location.
+ *
+ * https://www.bignerdranch.com/blog/web-accessibility-skip-navigation-links/
+ */
+
+
+export function init() {
```

Comment:
nochmal mötzle auf hochem niveau: das Skiplink-Züg aus `init` rausnehmen und in eine eigene Funktion tuten? Die könnte man dann ja auch separat aufrufen (wenn einem danach steht).

Reply:
voll gmacht!